languages/java/highlights.scm

```diff
@@ -289,7 +289,7 @@
   [
     "\\{"
     "}"
-  ] @string.special.symbol)
+  ] @punctuation) @embedded
```

Comment:
I would think that `@string.special.symbol` would be more accurate and be similar to the theming of string escapes, which makes sense to me, but if that's not how other Zed extensions conventionally do it then I agree, I wouldn't want to break consistency.

Reply:
I think `string.special.symbol` makes sense for escapes, and `punctuation` makes sense for interpolations. I omitted `special` but I can add it as other grammars do.

Reply:
Sure, go ahead and do `@punctuation.special`.